components/cam/src/physics/cam/physpkg.F90

```diff
@@ -39,8 +39,13 @@ module physpkg
   use cam_logfile,     only: iulog
   use camsrfexch,      only: cam_export
 
-  use modal_aero_calcsize,    only: modal_aero_calcsize_init, modal_aero_calcsize_diag, modal_aero_calcsize_reg
-  use modal_aero_wateruptake, only: modal_aero_wateruptake_init, modal_aero_wateruptake_dr, modal_aero_wateruptake_reg
+  use modal_aero_calcsize,    only: modal_aero_calcsize_init, &
+                                    modal_aero_calcsize_diag, &
+                                    modal_aero_calcsize_reg, &
+                                    modal_aero_calcsize_sub
+  use modal_aero_wateruptake, only: modal_aero_wateruptake_init, &
+                                    modal_aero_wateruptake_dr, &
```

Comment:
Since wateruptake_dr has been moved, we do not need it here in this use statement.

Reply:
thanks, fixed.